hoover/search/views.py

```diff
@@ -72,7 +95,6 @@ def search(request):
             highlight=body.get('highlight'),
             collections=[c.name for c in collections],
         )
-        success = True
```

Comment:
I think it's a good idea to keep `success = True` here. It means "we got to this point in the function, so we reached success". Otherwise, reading the code, you need to think about what `_search` returns, and it gets complicated.